service-commands/scripts/bake-dev-image.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# bakes from latest master on gcp by default
+
+set -x
+
+# error handling
+trap handle_provisioning_error 1
+handle_provisioning_error () {
+    "Error while provisioning - make sure cloud platform and service-commands A are set up. See service-commands README for details."
+    exit 1
+}
+
+# Prerequisites:
+    # service-commands installed w A alias
+    # GCP access (if baking w GCP)
+    # Azure access (if baking w Azure)
+# Default values of arguments
+USE_AZURE=0 # GCP by default
+SOURCE_DISK="christine-new-bake" # name of remote box to bake as AMI
+PROTOCOL_GIT_REF="master"
+CLIENT_GIT_REF="master"
+USERNAME="ubuntu"
+d=`date +%m-%d-%Y`
+echo $d
+BUILD_INSTANCE_NAME="audius-bake-dev-$d"
+
+process_arguments () {
+    for arg in "$@"
+    do
+        case $arg in
+            -a|--azure)
+            USE_AZURE=1
+            shift # Remove --azure from processing
+            ;;
+            -s|--source)
+            SOURCE_DISK="$2"
+            shift # Remove --source= from processing
+            shift
+            ;;
+            -p|--protocol)
+            PROTOCOL_GIT_REF="$2"
+            shift # Remove argument name from processing
+            shift # Remove argument value from processing
+            ;;
+            -c|--client)
+            CLIENT_GIT_REF="$2"
+            shift # Remove argument name from processing
+            shift # Remove argument value from processing
+            ;;
+        esac
+    done
+    echo "# Use Azure instead of GCP: $USE_AZURE"
+    echo "# Source disk: $SOURCE_DISK"
+    echo "# Protocol git ref to build at: $PROTOCOL_GIT_REF"
+    echo "# Client git ref to build at: $CLIENT_GIT_REF"
+}
+
+create_image_with_gcp() {
+    local GOOGLE_PROJECT_NAME="audius-infrastructure"
+    local DEFAULT_GCP_REGION="us-central1-a"
```

Comment:
I think it's `1f` by default. Cheran mentioned some docs were wrong.

Reply:
ah we decided to move to 1a, but it was pretty recent @cheran-senthil